connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -435,7 +445,9 @@ public Admin createAdminClient(Properties adminClientConfig) {
             adminClientConfig.put(SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG, ((Password) brokerConfig.get(SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG)).value());
             adminClientConfig.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, "SSL");
         }
-        return Admin.create(adminClientConfig);
+        Properties finalAdminConfig = Utils.mkProperties(additionalClientConfigs);
```

Comment:
In `createConsumer()` and `createProducer()` we add the custom client config first, here we do it the other way around. Let use similar logic if we can

Reply:
I wonder if we could also change the type of `adminClientConfig` to `Map` so it's similar to the other create methods.

Reply:
unified the logic for `createConsumer`, `createProducer` and `adminClientConfig` including changing type of `adminClientConfig` to `Map`